modules/widgets/milestone/class-milestone-widget.php

```diff
@@ -1,28 +1,34 @@
 <?php
-/*
-Plugin Name: Milestone
-Description: Countdown to a specific date.
-Version: 1.0
-Author: Automattic Inc.
-Author URI: https://automattic.com/
-License: GPLv2 or later
-*/
+/**
+ * Milestone Countdown Widget
+ *
+ * @package Jetpack.
+ */
 
 use Automattic\Jetpack\Assets;
 
+/**
+ * Registers the widget for use!
+ */
 function jetpack_register_widget_milestone() {
 	register_widget( 'Milestone_Widget' );
 }
 add_action( 'widgets_init', 'jetpack_register_widget_milestone' );
```

Comment:
Only within Jetpack, I'd agree. `widgets/milestone.php` isn't loaded on WP.com—only this file, so it would be a bit of a bigger refactor. I'm happy to do that, but would it be okay to do in a follow-up PR?

Reply:
Ok. But then (for my own edification) how does WP.com not get confused by the rename of the file?

Reply:
On the WP.com side, I did a number of commits:
0. Updated the Fusion filelist to look at new file name.
1. Copied the file to the new name and added `function_exists` check to ensure we wouldn't have duplicated definitions during the non-atomic deploy.
2. Updated the inclusion in the widget loader to point to the new file.
3. At this point, now the new file is loading exclusively, so deleted the old file and remove the function_exists check.